src/PaperG/FirehoundBlob/Facebook/Creative/FacebookCarouselCreativeData.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace PaperG\FirehoundBlob\Facebook\Creative;
+
+use PaperG\FirehoundBlob\Facebook\FacebookCreativeData;
+use PaperG\FirehoundBlob\Utility;
+
+class FacebookCarouselCreativeData extends FacebookCreativeData
+{
+    use Utility;
+
+    const PRIMARY = "primary";
+    const MULTI_SHARE_END_CARD = "multiShareEndCard";
+    const MULTI_SHARE_OPTIMIZED = "multiShareOptimized";
```

Comment:
enum, when we have time.  Would be a new version though, and break all the things.